lib/repl.js

```diff
@@ -1857,7 +1830,7 @@ ObjectSetPrototypeOf(Recoverable, SyntaxError);
 module.exports = {
   start,
   writer,
-  REPLServer,
+  REPLServer: function(...args) { return new REPLServer(...args); },
```

Comment:
https://github.com/nodejs/node/blob/daea065f249648a005a9caaa548a76db42eacbb1/doc/contributing/primordials.md#L261-L269
```suggestion
  REPLServer: function(...args) { return ReflectConstruct(REPLServer, args, new.target || REPLServer); },
```